src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -100,11 +150,45 @@ private void bindListeners(ReadOnlyPerson person) {
     }
 
     /**
-     * Changes the star metaphor image for each {@code Person} according to their favorite status
+     * Adds a profile photo for each {@code person}.
+     * TODO: This method will be modified for upcoming addPhoto command
+     */
+    private void initProfilePhoto (ReadOnlyPerson person) {
+        /* Round profile photo */
```

Comment:
Just curious, why use the multiline comment syntax for a single line comment?

Reply:
My bad, it's one of my bad habits~ Updated! 👍✔